tool/src/main/java/org/wildfly/security/tool/ElytronToolMessages.java

```diff
@@ -422,6 +451,27 @@
             "Blocks of options must be separated by a blank line.")
     String cmdFileSystemRealmBulkConvertDesc();
 
+    @Message(id = NONE, value = "Bulk conversion with options listed in description file. Optional options have default values, required options do not. (Action) %n" +
+            "The options realm-name, hash-charset, hash-encoding, and levels are optional. %n" +
+            "The options credential-store and secret-key are also both optional, but they must either both be unspecified or both be specified. %n" +
+            "The default values of realm-name, hash-charset, hash-encoding, and levels are encrypted-filesystem-realm, BASE64, UTF-8, and 2, respectively. %n" +
+            "If credential-store and secret-key are unspecified, they are set to \"mycredstore.cs\" and \"key\" %n" +
```

Comment:
This text needs to be updated since this is no longer the case.

Reply:
The new create option should be mentioned somewhere here.

Reply:
Looking closer, it seems like you'd also need an optional `populate` param that indicates whether the alias specified by `secret-key` should be automatically generated if it doesn't exist. It can default to true. Info about this new param would also need to be added to the description.

Reply:
The default value for `secret-key` can be `key`.